src/main/java/main/Main.java

```diff
@@ -0,0 +1,52 @@
+package main;
```

Comment:
The `main` package should be renamed to reflect the application's domain, such as `jv.fruit.shop` to keep consistency and avoid using a too generic package name.